### PR TITLE
Checking beneficiary address before depositing funds

### PR DESCRIPTION
**Pull request by dimpar**
Closes https://github.com/keep-network/keep-ecdsa/issues/491

Blocking unbonded value deposits when the beneficiary is not yet defined (delegation does not exist).
Removing check from `withdraw` function that previously blocked withdrawal if the beneficiary was not defined.

### CODE REVIEW

File: solidity/contracts/KeepBonding.sol [hunk 1]
```diff
@@ -314,6 +327,7 @@ contract KeepBonding {
 
         lockedBonds[bondID] = lockedBonds[bondID].sub(amount);
 
+        /* solium-disable-next-line */
```
Comment (pdyraga) [inline solidity/contracts/KeepBonding.sol:330]: Isn't it disabled in the global `.soliumrc.json`?

File: solidity/test/BondedECDSAKeepFactoryTest.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
   const notMember = accounts[5]
 
   const keepOwner = accounts[6]
+  const beneficiary = notMember
```
Comment (pdyraga) [inline solidity/test/BondedECDSAKeepFactoryTest.js:46]: Why don't we use `accounts[7]`?

File: solidity/test/BondedECDSAKeepTest.js [hunk 3]
```diff
@@ -1865,6 +1865,11 @@ contract("BondedECDSAKeep", (accounts) => {
   }
 
   async function depositForBonding(member1Value, member2Value, member3Value) {
+    const beneficiary = accounts[4]
```
Comment (pdyraga) [inline solidity/test/BondedECDSAKeepTest.js:1868]: Can we define `beneficiary` on the top of the file, where we define the `bondCreator`, `members`, `authorizers` and all the rest of the roles?

File: solidity/contracts/KeepBonding.sol (hunk 1 above)
Comment (dimpar) [reply solidity/contracts/KeepBonding.sol]: Yes, you're right. My VCS underlined it red and that mislead me.

File: solidity/test/BondedECDSAKeepFactoryTest.js (hunk 2 above)
Comment (dimpar) [reply solidity/test/BondedECDSAKeepFactoryTest.js]: No particular reason, just picked an account that not a member. Changed to accounts[7]

File: solidity/test/BondedECDSAKeepTest.js (hunk 3 above)
Comment (dimpar) [reply solidity/test/BondedECDSAKeepTest.js]: Sure, done.

**Pull request merged**